ui/components/PublishModal.js

```diff
@@ -40,15 +39,20 @@ export default function PublishModal(props) {
         "type" : "array",
         "items" : {
           "enum" : [
+            "Kubernetes",
+            "Argo CD",
+            "AWS App Mesh",
+            "Consul",
+            "Fluentd",
```

Comment:
Fluentd is named as fluentd-operator in the meshmodels fs structure. Also is app-mesh in the meshmodel the same as AWS App mesh here ? // @leecalcote

Reply:
@Avi-88 I think you have the right initial perspective: for lack of a better solution, let’s use the MeshModel model names as the sources of truth (use the MeshModel filesystem as the source).